Core/Utilities/RabbitMQAdmin.py

```diff
@@ -0,0 +1,134 @@
+"""RabbitMQAdmin module serves for the management of the internal RabbitMQ
+   users database. It uses rabbitmqctl command. Only the user with the right
+   permissions can execute those commands.
+"""
+import subprocess
+import re
+from DIRAC import S_OK, DError
+import errno
+
+def executeRabbitmqctl(arg, *argv):
+  """Executes RabbitMQ administration command.
+    It uses rabbitmqctl command line interface.
+    For every command the -q argument ("quit mode")
+    is used, since in some cases the output must be processed,
+    so we don't want any additional informations printed.
+  Args:
+    arg(str): command recognized by the rabbitmqctl.
+    argv(list): optional list of string parameters.
+  Returns:
+    S_OK:
+    Derror:
+
+  """
+  command =['sudo','/usr/sbin/rabbitmqctl','-q', arg] + list(argv)
+  res = subprocess.Popen(command, stdout = subprocess.PIPE, stderr= subprocess.PIPE)
```

Comment:
I changed it to systemCall,
no Idea what timeout to set.
Any suggestions, appreciated.